api/PeriodicSyncEvent.json

```diff
@@ -34,10 +35,10 @@
             "version_added": false
           },
           "samsunginternet_android": {
-            "version_added": false
+            "version_added": true
```

Comment:
Where did the Samsung Internet data come from? I'm not saying it's wrong, but it doesn't exactly square with the browser version mappings we have in `/browsers/samsung_internet.json`.

The Chrome, WebView, and MDN URLs look fine, so this is the only part I'm concerned about.

Reply:
This is me forgetting that Samsung doesn't fork the current version of Chrome. I've changed this to false everywhere.

Reply:
Looks like a few entries are still left as `true` in PeriodicSyncManager?

Reply:
Sorry about that. I think I got them this time.

Reply:
Not a problem, looks good now!